libarchive/archive_write_set_format_7zip.c

```diff
@@ -755,14 +755,19 @@ _7z_close(struct archive_write *a)
 		 */
 #if HAVE_LZMA_H
 		header_compression = _7Z_LZMA1;
+        if(zip->opt_compression == _7Z_LZMA2 ||
```

Comment:
Libarchive sources use tabs for indentation; please be consistent.

Reply:
thank you, fixed.